.gitmodules

```diff
@@ -1,3 +1,4 @@
 [submodule "external/monero-cpp"]
 	path = external/monero-cpp
-	url = https://github.com/woodser/monero-cpp
+	url = https://github.com/nsec1/monero-cpp.git
+	branch = buildStaticOption
```

Comment:
Can this be removed? It's not checking out this branch, and the build is working without it.

Reply:
Yes, it can. It just makes tests easier. I will do this.